packages/pds/src/api/com/atproto/admin/sendEmail.ts

```diff
@@ -28,11 +29,13 @@ export default function (server: Server, ctx: AppContext) {
         return resultPassthru(
           await ctx.entrywayAgent.com.atproto.admin.sendEmail(
             input.body,
-            await ctx.serviceAuthHeaders(
-              recipientDid,
-              ctx.cfg.entryway.did,
-              ids.ComAtprotoAdminSendEmail,
-            ),
+            await ctx
+              .serviceAuthHeaders(
+                recipientDid,
+                ctx.cfg.entryway.did,
+                ids.ComAtprotoAdminSendEmail,
+              )
+              .then((x) => forwardIp(req, x)),
```

Comment:
Would it make sense to add a special `ctx.entrywayServiceAuthHeaders(req, did, lxm)` method that wraps this ? It makes it less likely, in the future, that someone forgets to forward the ip when proxying fo entryway, don't you think ?

Reply:
The following lines, for example, should probably be using the same logic:

https://github.com/bluesky-social/atproto/blob/75c05d50b6a7283555f01eb783fc968b29d62d6b/packages/pds/tests/entryway.test.ts#L120-L124

Reply:
Yupyup good call, should be set now.